.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "[javascript][typescript][typescriptreact]": {
+    "editor.formatOnSave": false,
```

Comment:
I use `true` and use prettier as my default formatter.
How about not putting `.vscode` folder into git, so people can use different editors?
I'm not sure what the trend is nowadays.

Reply:
Since workspace settings take priority, it might be better not to include it.